string axis #all,x,y,z
